### PR TITLE
- added general design

### PR DESCRIPTION
**Pull request by ipopa28**
- added logo, icons, fonts, colours, components

### CODE REVIEW

**Comment by TBD** [comment]
- [ ] add package.json data for `react-bootstrap` and `bootstrap`
- [ ] add screenshot of components in action (add to a readme.md in design subfolder, see markdown docs)
- [ ] add a step-by-step guide on how/when to use the components

File: frontend/src/App.css [hunk 1]
```diff
@@ -1,38 +1,83 @@
...
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
+.formRange{
+  width:31%;
 }
```
Comment (VladTM23) [inline frontend/src/App.css:83]: Are all "!important" tags necessary?

I'm just asking if there may be other way of overriding Bootstrap styling, don't really know.

File: frontend/src/App.js [hunk 2]
```diff
@@ -1,26 +1,143 @@
...
+
+        </div>
+
+      </div>
+
+    );
+  }
 }
 
 export default App;
```
Comment (VladTM23) [inline frontend/src/App.js:143]: App needs to be a functional component (not class component in order for hooks & custom hooks to work). 

Refactor that Bootstrap styling either in index.css, or App.css or for every particular component (for example, a Button component, create a simple Component of your own, import Bootstrap button there and add a .css file for your own component and only import your own component).

File: design/Logo/Logo_vertical.svg [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+<feFlood flood-opacity="0" result="BackgroundImageFix"/>
+<feColorMatrix in="SourceAlpha" type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"/>
+<feOffset dy="4"/>
+<feGaussianBlur stdDeviation="2"/>
+<feColorMatrix type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.25 0"/>
+<feBlend mode="normal" in2="BackgroundImageFix" result="effect1_dropShadow"/>
+<feBlend mode="normal" in="SourceGraphic" in2="effect1_dropShadow" result="shape"/>
+</filter>
+</defs>
+</svg>
```
Comment (VladTM23) [inline design/Logo/Logo_vertical.svg:19]: This is a GENERAL comment for all the stuff in the design folder. 

You already have them all in frontend, so I think we do not need them in the design folder anymore (or we could have the design folder in the frontend folder).

**Pull request merged**